hub/@tldraw/drawing-canvas.json

```
@@ -0,0 +1,6 @@
+{
+  "repository": "https://github.com/hashintel/hash.git",
+  "commit": "6a8932a439077e270fbb2945800981b9305f5b3d",
```

[This commit](https://github.com/hashintel/hash/commit/6a8932a439077e270fbb2945800981b9305f5b3d) is not on a main branch, let’s update it to, say:

```suggestion
  "commit": "70db9afee5cc5ce3c0e39af14f2699f0cf031963",
```

([the latest one ATM](https://github.com/hashintel/hash/commits/main))

---

Updated in [2178e23](https://github.com/blockprotocol/blockprotocol/pull/316/commits/2178e2318b1f18005f7a157822fbaa7d736a9ea0)